doc/manual/manual.tex

```diff
@@ -1939,10 +1939,23 @@ \subsubsection{The isothermal compression approximation (ICA)}
 so we get
 \begin{equation}
 \label{eq:stokes-2-compressible}
-\nabla \cdot \textbf{u} = -\frac{1}{\rho} \frac{\partial \rho}{\partial p} \rho \textbf{g} \cdot \textbf{u}
+\nabla \cdot \textbf{u} = - \rho \frac{1}{\rho} \frac{\partial \rho}{\partial p} \textbf{g} \cdot \textbf{u}
 \end{equation}
 where $\frac{1}{\rho} \frac{\partial \rho}{\partial p}$ is often referred to
-as the compressibility.
+as the compressibility. Note that we have not yet made any assumptions about the
+change in temperature with pressure; we need to do this in order to calculate the
+compressibility. There are two simple choices we could make; either to ignore
+adiabatic heating and use the isothermal compressibility:
+\begin{equation}
+\nabla \cdot \textbf{u} = - \rho \beta_T \textbf{g} \cdot \textbf{u}
```

Comment:
I feel this would be even more instructive if you could add a step in between where you add the index T or S, basically `\nabla \cdot \textbf{u} = -\frac{1}{\rho} \frac{\partial \rho}{\partial p}_T \rho \textbf{g} \cdot \textbf{u} = - \rho \beta_T \textbf{g} \cdot \textbf{u}`